CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+## Adding new pages :
+
+The single page tree structure is centralized in the following file : [`src/utils/pageConfig.tsx`](src/utils/pageConfig.tsx).
```

Comment:
I think the link to the file is incorrect now - routePathsConfig

Reply:
will be corrected thx